packages/website/containers/Layout.tsx

```diff
@@ -1,13 +1,23 @@
+import { useNearHooksContainer } from '@cura/hooks'
+import { Box, Container, Spinner } from 'theme-ui'
+
 import Header from './Header'
 
 
+export default function Layout({ children } : {children: JSX.Element}){
 
-export default function Layout(){
+	const { accountId } = useNearHooksContainer();
 
 
 	return(
 		<>
 			<Header />
+
+			<Container >
+                { accountId && (
```

Comment:
We don't want to require login for all pages. 

We could add a prop to Layout called `requireAuth` so we can choose which page needs login